classifier/utils.py

```diff
@@ -40,77 +40,31 @@ def get_token_list(text):
   return text.split()
 
 
-def yield_sources_and_targets(
-    input_file,
-    input_format):
+def yield_sources_and_targets_meaning(input_file):
   """Reads and yields source lists and targets from the input file.
 
   Args:
     input_file: Path to the input file.
-    input_format: Format of the input file.
 
   Yields:
     Tuple with (list of source texts, target text).
```

Comment:
A tuple of (list of source texts, target text, score).

Reply:
Done